### PR TITLE
[DOC] fix docstring

### PR DESCRIPTION
**Pull request by blaggacao**
self explanatory

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: odoo/models.py [hunk 1]
```diff
@@ -1468,7 +1468,7 @@ def search(self, args, offset=0, limit=None, order=None, count=False):
         :param int offset: number of results to ignore (default: none)
         :param int limit: maximum number of records to return (default: all)
         :param str order: sort string
-        :param bool count: if True, only counts and returns the number of matching records (default: False)
+        :param bool count: if True, returns a list of ids. Internal use only. (default: False)
```
Comment (yelizariev) [inline odoo/models.py]: I'd replace `` Internal use only.`` with `` Internal use only, use search_count instead``

**Comment by xmo-odoo** [comment]
This change is not correct, because if it's passed `count=True` (and unless mis-overridden) `BaseModel._search` will return an integer: https://github.com/odoo/odoo/blob/379b7359621fd1239b0b4ae9f1fbc2954e574e95/odoo/models.py#L3772-L3778 Hence the special case: we don't want to wrap that integer in a recordset as it makes no sense.

**Comment by yelizariev** [comment]
You are right. I think this line was the reason of confusion 

https://github.com/odoo/odoo/blob/379b7359621fd1239b0b4ae9f1fbc2954e574e95/odoo/models.py#L1477

i.e. res when ``count`` is True and res when ``count`` is False is different *type* of result

**Comment by xmo-odoo** [comment]
Indeed. That's one of the historical... misfeatures of the system.

**Pull request closed without merging**